src/api/Message.js

```diff
@@ -17,10 +17,11 @@ export function getMessages(ticker) {
  * @param {string} text
  * @returns {Promise<Response>}
  */
-export function postMessage(ticker, text) {
+export function postMessage(ticker, text, geoInformation) {
     return Auth.fetch(`${ApiUrl}/admin/tickers/${ticker}/messages`, {
         body: JSON.stringify({
-            text: text
+            text: text,
+            geoInformation: geoInformation
```

Comment:
I used underscore notation for keys inside the api.
```suggestion
            geo_information: geoInformation
```